DOCS-7912 made edits (#77)
mulesoft/docs-service-mesh: modules/ROOT/pages/index.adoc

:keywords: service mesh, microservices, cluster, container

> Anypoint Service Mesh enables you to extend your microservices network by including your non-MuleSoft applications into the Anypoint Platform sphere. You can then manage and secure your application network, including your non-Mule applications, seamlessly from a single pane.

Reviewer: I wonder if it is okay to use "the Anypoint Platform sphere"? Maybe "your Anypoint Platform environment"?

> Your non-MuleSoft microservices might be written using different languages and platforms, or might be deployed in different environments. To alleviate issues arising from differences in these microservices, Anypoint Service Mesh ensures the following application management advantages:

Reviewer: ```suggestion
Your non-MuleSoft microservices might be written using different languages and platforms or might be deployed in different environments. To alleviate issues arising from differences in these microservices, Anypoint Service Mesh provides the following application-management advantages:
```
There is a legal reason that we have to avoid saying that a product "ensures" anything, as if it were a promise or guarantee.

* Provides resilient service-to-service communication using zero-trust policies to handle traffic control, fault tolerance and load balancing 
* Extends MuleSoft API Management capabilities to any non-Mule service, with policies, security and analytics all applied directly from Anypoint Platform

> * Amplifies reuse of services with discovery of non-Mule services within Exchange, available for use in future projects

Reviewer: * Resilient...
* Extended...
* Increased...

== Anypoint Service Mesh Architecture

> A service mesh is an independent architecture layer, which is encapsulated in a Kubernetes Cluster. In a service mesh architecture, instead of the services directly communicating with one another, a sidecar proxy is used to perform the same job.  

Reviewer: ```suggestion
A _service mesh_ is an independent architecture layer within a Kubernetes cluster. In a service mesh architecture, services communicate with one another via a sidecar proxy rather than directly.  
```

> The sidecar proxy enables you to bring your existing non-Mule microservices from the Kubernetes Cluster as APIs into Anypoint Platform. You can then manage these APIs within API Designer, API Manager, and Exchange. Additionally, you can monitor the health of your APIs using API analytics.

Reviewer: ```suggestion
The sidecar proxy enables you to bring your existing non-Mule microservices from the Kubernetes cluster into Anypoint Platform as APIs. You can then manage these APIs within API Designer, Anypoint API Manager, and Anypoint Exchange. Additionally, you can monitor the health of your APIs using API analytics.
```
Do you mean Anypoint Analytics, the product?

> A mixer, whose key function is to provide a layer of abstraction over the different policy and telemetry backend systems, governs the sidecar proxies. Because of this abstraction, applications such as Anypoint Platform, enable the rest of Istio to be agnostic of the backend processing.

Reviewer: ```suggestion
A _mixer_, whose key function is to provide a layer of abstraction over the different policy and telemetry backend systems, governs the sidecar proxies. Because of this abstraction, applications such as Anypoint Platform enable the rest of Istio to be agnostic of the backend processing.
```

> The mixer is architected on a general-purpose plug-in model, which provides flexibility in dealing with different infrastructure backend applications. These plug-in applications, known as adapters, enable the mixer to interface with the backend applications that deliver core functionality, such as logging, monitoring, enforcing quotas, performing ACL checks, and so on.

Reviewer: ```suggestion
The mixer follows a general-purpose, plug-in model, which provides flexibility in working with different infrastructure backend applications. These plug-in applications, known as _adapters_, enable the mixer to interface with the backend applications that deliver core functionality, such as logging, monitoring, enforcing quotas, performing ACL checks, and so on:
```
Does the mixer plug in to the applications, or vice versa? Consider a simpler rewording, like "the mixer plugs in to different backend applications, known as _adapters_, so that it can work with them to deliver core....

> image::anypoint-service-mesh-architecture.png[Anypoint Service Mesh architecture]

Reviewer: At some point, you should look in to providing ALT text for all of your images. Hanna is a good resource for learning how to do this, and I believe that it is going to be required for all of our graphic elements, if it is not already.

. The mixer communicates the outcome of the policy checks to the Envoy proxy.
. When no policy violations are encountered, the request is routed to the microservice. The microservice runs the service logic and sends the response back to the client.
. The adapter communicates with Anypoint Platform asynchronously to get the latest policies and contracts.

> . Finally, the adapter sends back the API analytics information to Anypoint Platform.

Reviewer: ```suggestion
. Finally, the adapter returns the API analytics information to Anypoint Platform.
```
